### PR TITLE
fix: Disable moka cache by default

### PR DESCRIPTION
**Pull request by Xuanwo**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

fix: Disable moka cache by default

### CODE REVIEW

File: src/query/config/src/outer_v0.rs [hunk 1]
```diff
@@ -518,6 +518,9 @@ impl From<InnerCacheConfig> for CacheConfig {
         };
 
         match inner.params {
+            StorageParams::Disabled => {
+                cfg.cache_type = "disabled".to_string();
```
Comment (BohuTANG) [inline src/query/config/src/outer_v0.rs:522]: It is better to add a new cache type 'None' or `BlackHole`. It means we can write through this cache backend, the cache layer does nothing except we configure the cache.

File: src/common/storage/src/operator.rs [hunk 2]
```diff
@@ -72,6 +75,12 @@ pub fn init_operator(cfg: &StorageParams) -> Result<Operator> {
         StorageParams::S3(cfg) => init_s3_operator(cfg)?,
         StorageParams::Oss(cfg) => init_oss_operator(cfg)?,
         StorageParams::Redis(cfg) => init_redis_operator(cfg)?,
+        _ => {
+            return Err(Error::new(
+                ErrorKind::InvalidInput,
+                anyhow!("disabled is not a valid storage type"),
```
Comment (BohuTANG) [inline src/common/storage/src/operator.rs:81]:
```suggestion
                anyhow!("Unuppport storage type: {{}"),
```

File: src/common/storage/src/operator.rs [hunk 3]
```diff
@@ -72,6 +75,12 @@ pub fn init_operator(cfg: &StorageParams) -> Result<Operator> {
         StorageParams::S3(cfg) => init_s3_operator(cfg)?,
         StorageParams::Oss(cfg) => init_oss_operator(cfg)?,
         StorageParams::Redis(cfg) => init_redis_operator(cfg)?,
+        _ => {
+            return Err(Error::new(
+                ErrorKind::InvalidInput,
+                anyhow!("Unuppport storage type: {{}"),
```
Comment (Xuanwo) [inline src/common/storage/src/operator.rs:81]:
```suggestion
                anyhow!("Unuppport storage type"),
```

File: src/common/storage/src/operator.rs [hunk 4]
```diff
@@ -72,6 +75,12 @@ pub fn init_operator(cfg: &StorageParams) -> Result<Operator> {
         StorageParams::S3(cfg) => init_s3_operator(cfg)?,
         StorageParams::Oss(cfg) => init_oss_operator(cfg)?,
         StorageParams::Redis(cfg) => init_redis_operator(cfg)?,
+        _ => {
+            return Err(Error::new(
+                ErrorKind::InvalidInput,
+                anyhow!("Unuppport storage type"),
```
Comment (BohuTANG) [inline src/common/storage/src/operator.rs]: Typo: `Unuppport` :/
We can bring the storage type to this error message.

**Review by BohuTANG — APPROVED** [review]

**Review by zhyass — APPROVED** [review]
LGTM

**Pull request merged**